docs/content/developers/building-contributing.md

```diff
@@ -47,7 +47,7 @@ rm ddev-macos-arm64.zip
 
 ![Github Action PR Comment ZIP files](../images/github-action-pr-comment.png)
 
-Tip: If you need a zip-file to try out the "Testing a PR" process, see the [nightly builds](https://nightly.link/ddev/ddev/workflows/master-build/master).
+Tip: If you need a zip-file to try out the "Testing a PR" process, see the [nightly builds](https://nightly.link/ddev/ddev/workflows/master-build/main).
```

Comment:
Let's change the name of the master-build workflow too. It's a poor name anyway. But I guess `main-build` would be descriptive. Trying to think of something else. But the idea is "main branch build" as opposed to "PR build". I guess it could become "main-branch-build", and the name inside the yaml would change too.

Reply:
Agreed.